scipy/spatial/transform/rotation.py

```diff
@@ -1460,3 +1461,109 @@ def __getitem__(self, indexer):
                [ 0.57735027,  0.57735027, -0.57735027,  0.        ]])
         """
         return self.__class__(self._quat[indexer], normalized=True)
+
+    @classmethod
+    def match_vectors(cls, a, b, weights=None, normalized=False):
+        """Estimate the rotation to match two sets of vectors.
+
+        Find a rotation between frames A and B which best matches a set of unit
+        vectors `a` and `b` observed in these frames. The following loss
+        function is minimized to solve for the direction cosine matrix
+        :math:`C`:
+
+        .. math::
+
+            L(C) = \\frac{1}{2} \\sum_{i = 1}^{n} w_i \\lVert \\mathbf{a}_i -
+            C \\mathbf{b}_i \\rVert^2 ,
+
+        where :math:`w_i`'s are the `weights` corresponding to each vector.
+
+        The rotation is estimated using Markley's SVD method [1]_.
+
+        Parameters
+        ----------
+        a : array_like, shape (N, 3)
+            Vectors expressed in initial frame A. Result of applying an
+            arbitrary rotation to `b`. Each `a[i]` corresponds to a vector.
+        b : array_like, shape (N, 3)
+            Vectors expressed in another frame B. Shape must match `a`. Each
+            `b[i]` corresponds to a vector.
+        weights : None or array_like, 1D
+            Coefficients describing the relative importance of the vectors in
```

Comment:
I suggest to shorten it to "Weights describing relative importance of observations".  And keep the last sentence as `If None (default) ...".

Describe shape as `array_like, shape (N,) or None`.

Reply:
`None, optional` would be the way IIRC if there are optional args or  `(N,) array_like`